iamy/cfn.go

```diff
@@ -135,7 +136,7 @@ func (c *cfnClient) IsManagedResource(cfnType CfnResourceType, resourceIdentifie
 		return c.managedResources[resourceIdentifier].contains(cfnType)
 	}
 
-	if cfnResourceRegexp.MatchString(resourceIdentifier) {
+	if cfnResourceRegexp.MatchString(resourceIdentifier) && strings.ContainsAny(resourceIdentifier, UpperCaseLetters) {
```

Comment:
We could limit this to the last component of the resourceIdentifier (after the last `-`). It probably doesn't make a difference for the pattern we're looking to exclude though.

Reply:
I think it will sadly, I've pushed a fix and a couple of tests to catch it.

Reply:
https://github.com/envato/iamy/pull/11/commits/5635ecd4361b146e4d365ca9c67243e795db3107